docs/02-Deploying/06-Upgrading-Fleet.md

```diff
@@ -0,0 +1,58 @@
+# Upgrading Fleet
+
+This guide explains how to upgrade your Fleet instance to the latest version in order to get the latest features and bug fixes. For initial installation instructions, see [Installing Fleet](./02-Server-Installation.md).
```

Comment:
Should "Installing Fleet" link to [server-installation#installing-fleet?](https://fleetdm.com/docs/deploying/server-installation#installing-fleet)?

Reply:
Perhaps I'm wrong here, because a user may wish to do a server installation?